src/testing.py

```diff
@@ -23,6 +23,7 @@
 
 import proc
 
+SINGLE_THREADED = False
```

Comment:
Can you pipe this in from the command line instead?

Reply:
Done... although still using a global.. otherwise we need to patch all the callsites..